.github/workflows/unittests.yml

```diff
@@ -25,10 +25,10 @@ jobs:
           conda --version
           which python
           pip install numpy cython
-          pip install -e .
           pip install pytest">=3.6" coverage pytest-cov coveralls
       - name: Run unit tests
         run: |
+          pip install -e .
```

Comment:
Not sure why you change this?
If no particular reason, I would keep the `install` in the `Installation` step.

Reply:
I thought of installation step as installation of dependencies. 
If you prefer the previous version, I'll revert that change.

Reply:
Let's keep like this then :)
Not really a big deal.